test/integration.test.js

```diff
@@ -0,0 +1,21 @@
+// Using mocha, since for memory leak testing we want the tests run serially, anyway
+
+const expect = require('chai').expect
+const leakage = require('../lib/index')
+
+describe('leakage', () => {
+  it('throws an error when testing leaky code', () => {
+    const objects = []
+
+    expect(() => leakage.iteration(100, () => {
```

Comment:
3945066a3ad4de91976acd20886e41782036acd1 later in this PR merges the change that renames `iteration` -> `iterate`, but this test is never updated to the new name, so it's only the deprecated export that's being tested.